airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ReplicationJobOrchestrator.java

```diff
@@ -89,6 +107,25 @@ public Optional<String> runJob() throws Exception {
         sourceLauncherConfig.getDockerImage().equals(WorkerConstants.RESET_JOB_SOURCE_DOCKER_IMAGE_STUB) ? new EmptyAirbyteSource()
             : new DefaultAirbyteSource(workerConfigs, sourceLauncher);
 
+    final FeatureFlags featureFlags = new EnvVariableFeatureFlags();
```

Comment:
the job orchestrator runs in the `jobs` namespace as part of a job in Cloud. in theory, the orchestrator is fire-and-forget, so I don't think we want to allow direct database access from the orchestrator. Doing so also present some security risk, since today we sandbox the `jobs` namespace off from the `ab` namespace, so we would have to make some allowances for the orchestrator pod - not the worst but not very clean.

I think the right way to do this is to inject the workspace id via the ReplicationActivityImpl (this runs in the `ab` namespace). This follow how configs are currently propagated to the jobs - as static files - so we can keep the interface consistent.